exercises/atbash-cipher/cipher.t

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env perl6
+
+use Test;
+use JSON::Tiny;
+
+use lib ( my $dir = IO::Path.new($?FILE).parent ).path;
+
+my $module_name = %*ENV<EXERCISM>.so ?? 'Example' !! 'Allergies';
+my @potential_module = <p6 pm6 pm>.map:  $module_name ~ '.' ~ *; 
+
+my $module = first { $dir.child($_).e }, |@potential_module
+    or die "No file '$module_name.p6' found\n";
+
+require $module <&encode &decode>;
+
+plan 2;
+
+my %cases = from-json $dir.child('cases.json').slurp;
+
+subtest 'encode' => {
+    my @cases = |%cases.<encode>.<cases>;
+
+    plan +@cases;
+
+    is encode( .<phrase> ), .<expected>, .<description>
+        for @cases;
+};
+
+subtest 'decode' => {
+    my @cases = |%cases.<decode>.<cases>;
+
+    plan +@cases;
+
+    is decode( .<phrase> ), .<expected>, .<description>
+        for @cases;
+};
+
+done-testing;
```

Comment:
`done-testing` here is akin to `return` as the last statement of a function. Perl would return the value yielded by the last statement with or without the presence of the keyword `return`, but its presence is a visual cue to the reader that the function is expected to return/end at this point. It is also more than what a comment would provide as it also acts as a sanity check that will dot and cross all the TAP Is and Ts.

This being said, while I don't feel like changing the code for something I feel is spurious, I won't mind if you do so alter the branch yourself. Go wild. :-)